user/architecture.rst

```diff
@@ -0,0 +1,27 @@
+Architecture
+============
+
+The following image summarizes the OpenWISP architecture.
+
+.. image:: /assets/design/meta-openwisp-architecture.jpg
+   :align: center
+   :alt: OpenWISP Architecture Meta Level
+
+A `larger version of the architecture image <https://miro.com/app/board/o9J_kvt3edQ=/?moveToWidget=3074457361507412175&cot=14>`_ and more detailed architectures of the modules can be found on Miro.
+
+- Server Side:
+   - OpenWISP (You have 2 main options):
+       - Ansible-OpenWISP2 (Recommended): Primary method to install
+         OpenWISP. Find `ansible-openwisp2 architecture details on GitHub <https://github.com/openwisp/ansible-openwisp2#architecture>`_.
+       - Docker-OpenWISP (In-Alpha): The future for installation of
+         OpenWISP. Find `docker-openwisp architecture details on GitHub <https://github.com/openwisp/docker-openwisp#architecture>`_.
+   - WiFi Login Pages (Optional): Configurable captive page for
+     public/private WiFi services providing login, sign up etc.
```

Comment:
would be good to link https://github.com/openwisp/ansible-openwisp-wifi-login-pages

Reply:
Done.